app/assets/stylesheets/game_day.css.scss

```diff
@@ -0,0 +1,21 @@
+body#game_day {
```

Comment:
Avoid using id selectors<br>Avoid qualifying id selectors with an element.<br>Selector `game_day` should be written in lowercase with hyphens